### PR TITLE
Create README.md and use Gradient Boosting Classifier

### PR DESCRIPTION
**Pull request by shubhayansarkar**
# Gradient Boosting Classifier
from sklearn.ensemble import GradientBoostingClassifier

gbk = GradientBoostingClassifier()
gbk.fit(x_train, y_train)
y_pred = gbk.predict(x_val)
acc_gbk = round(accuracy_score(y_pred, y_val) * 100, 2)
print(acc_gbk)
accuracy===84.77
you can check it out

### CODE REVIEW

**Comment by arora1209** [comment]
yeah! it works .
Thanks alot!
